tests/Session/test_profile.py

```diff
@@ -35,7 +35,15 @@ def testOpen(self):
 
         # Should return 1 if a food storage exists.
         self.profile.addFoodStorage(FREEZER, freezer_name)
-        self.assertEqual(1, self.profile.open(FREEZER, freezer_name))       
+        self.assertEqual(1, self.profile.open(FREEZER, freezer_name)) 
+
+    def testGetOpenFoodStorage(self):
+        # Should return None if food storage is not open. 
+        self.assertIsNone(self.profile.getOpenFoodStorage())
+       
+        # Should return not none if food storage is open. 
+        self.profile.addFoodStorage(FREEZER, freezer_name)
+        self.assertIsNotNone(self.profile.getOpenFoodStorage())
```

Comment:
An additional check you can do is checking that it's of type Freezer. (See line 26 in this file)

Reply:
Done!